docs/developer-docs/security/rust-canister-development-security-best-practices.md

```diff
@@ -419,19 +428,19 @@ If a canister traps or panics in `pre_upgrade`, this can lead to permanently blo
 
 - Panic in the `post_upgrade` hook if state is invalid, so that one can retry the upgrade and try to fix the invalid state. Panics in the the post-upgrade hook abort the upgrade, but one can retry with new code.
 
-- [Test the upgrade hooks.](https://mmapped.blog/posts/01-effective-rust-canisters.html#test-upgrades) (from [Effective Rust Canisters](https://mmapped.blog/posts/01-effective-rust-canisters.html))
+- [Test the upgrade hooks](https://mmapped.blog/posts/01-effective-rust-canisters.html#test-upgrades) (from [effective Rust canisters](https://mmapped.blog/posts/01-effective-rust-canisters.html)).
 
-- See also the section on upgrades in [How to audit an Internet Computer canister](https://www.joachim-breitner.de/blog/788-How_to_audit_an_Internet_Computer_canister) (though focused on Motoko)
+- See also the section on upgrades in [how to audit an Internet Computer canister](https://www.joachim-breitner.de/blog/788-How_to_audit_an_Internet_Computer_canister) (though focused on Motoko).
 
-- See [Current limitations of the Internet Computer](https://wiki.internetcomputer.org/wiki/Current_limitations_of_the_Internet_Computer), section "Bugs in `pre_upgrade` hooks"
+- See [current limitations of the Internet Computer](https://wiki.internetcomputer.org/wiki/Current_limitations_of_the_Internet_Computer), section "Bugs in `pre_upgrade` hooks".
 
 ### Reinstantiate timers during upgrades
 
 #### Security Concern
 
 Global timers are deactivated upon changes to the canister's Wasm module. The [IC specification](../../references/ic-interface-spec#timer) states this as follows:
 
-> The timer is also deactivated upon changes to the canister's Wasm module (calling install_code, uninstall_code methods of the management canister or if the canister runs out of cycles). In particular, the function canister_global_timer won't be scheduled again unless the canister sets the global timer again (using the System API function ic0.global_timer_set).
+The timer is also deactivated upon changes to the canister's Wasm module (calling install_code, uninstall_code methods of the management canister or if the canister runs out of cycles). In particular, the function canister_global_timer won't be scheduled again unless the canister sets the global timer again (using the System API function ic0.global_timer_set).
```

Comment:
Only a minor thing: Here I'm quoting from the IC specification and thus use the Markdown Blockquote ">". Did you intentionally remove it to unify with other places? Or is there maybe still some way of showing this paragraph as a quote?

Reply:
I've pushed a change to reinstate the Blockquote with ">" and added quotation marks around the quote to further emphasis that this is a direct quote from the IC spec page.